modules/reliability/jsx/columnFormatter.js

```diff
@@ -0,0 +1,27 @@
+function formatColumn(column, cell, rowData, rowHeaders) {
+// If a column if set as hidden, don't display it
+  if (loris.hiddenHeaders.indexOf(column) > -1) {
+    return null;
+  }
+
+  // Create the mapping between rowHeaders and rowData in a row object.
+  var row = {};
+  rowHeaders.forEach(function(header, index) {
+    row[header] = rowData[index];
+}, this);
+
+ 
+    if(column === 'PSCID' ) {
+        if (row["Manual Swap"]==="yes")
+        {
```

Comment:
I can't remember if JS will implicitly add a semicolon before the `{` here or not ( @alisterdev? @jstirling91?)

Regardless, the formatting of this whole block is off and hard to read. Can you run this file through eslint and fix the formatting?